### ISSUE TITLE
Windows Signing

### ISSUE THREAD

**Issue by TheRealHypo** [edvin/fxlauncher#157]
Hey, I'm struggling for some hours now, to find a way to sign my Native Windows application created by FxLauncher.

Could anyone point me in the right direction ? Currently we are struggling, because a new AV in our company kills our scripts, because they are from "untrusted developers".

So how to Sign an fxLauncher Native? Do I have to sign the installer ? The Jar ? 

Is SignTool.exe from Microsoft the correct Tool ?

**Comment by edvin** [comment]
Yes, the easiest solution is simply to build the executable and sign it manually on the command line using signtool. All you need is in the signtool docs :)
